dev/sparktestsupport/modules.py

```diff
@@ -743,6 +743,20 @@ def __hash__(self):
     ],
 )
 
+pyspark_types = Module(
```

Comment:
I am kind of hesitant whether it is too much to add a new testing module.
maybe we can just add these tests in `pyspark.sql.tests.pandas`?

Reply:
I don't have a strong preference. @HyukjinKwon ?